tony-history-server/app/utils/SecurityUtils.java

```diff
@@ -0,0 +1,82 @@
+package utils;
+
+import com.typesafe.config.Config;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hdfs.HdfsConfiguration;
+import org.apache.hadoop.security.UserGroupInformation;
+
+import java.io.File;
+import java.io.IOException;
+import org.apache.hadoop.yarn.api.ApplicationConstants;
+import org.apache.hadoop.yarn.conf.YarnConfiguration;
+import play.Logger;
+
+/**
+ * The class handles authentication when cluster is security enabled
+ */
+public class SecurityUtils {
+  private static final Logger.ALogger LOG = Logger.of(SecurityUtils.class);
+  private static Config appConf;
+  private static FileSystem fs;
```

Comment:
This class never seems to use `fs`; it only creates and stores it. I think this should be moved out of `SecurityUtils` and only stored where it's needed. create/getting an FS should probably be done in HdfsUtils.